light_curves/light_curve_generator.md

```diff
@@ -349,6 +350,31 @@ df_lc_icecube = icecube_get_lightcurves(sample_table, icecube_select_topN=3)
 df_lc.append(df_lc_icecube)
 
 print('icecube search took:', time.time() - icecubestarttime, 's')
+```
+
+### 3.3 Rubin Data 
+Before running this section, you need to have both 1) a login to the Rubin Science Platform (RSP) and 2) an authenticating token setup in your Fornax home directory. 
+
+1) To see if you have Rubin data rights, and if you do, to get that login setup, follow these [instructions](https://rsp.lsst.io/guides/getting-started/get-an-account.html) 
+
+2) After logging in to RSP, follow these [instructions](code_src/rsp_token_instructions.txt) to get a token and store it in your home directory.
+
+
+This code will not work without the above information, so we have commented it out for now.  Uncomment when you are ready to proceed.
```

Comment:
maybe uncomment the whole cell, or change the cell type? 
(I think that maybe easier than figuring out the few lines peppered through that needed to be uncommented. OTOH, if it's peppered through the user may actually get some insight which line is doing the actual work 😅 )

Reply:
Done. Commented out the whole cell and added explicit instructions in that cell.  I figured changing cell type was just going to annoy either you, or someone else who doesn't use jupyter to do their work.  Commented lines seem a more universal way.

Reply:
nope, I'm OK with changing the type, but not sure which one is more user friendly.

Also, unrelated but this reminded me that we can hide cells or make them hidden (with an option to dropdown/extend) with metadata. I think at some point we can start using these features, if it makes sense to you.